### DISCUSSION TITLE
Are your zopen tools not being upgraded with the `zopen upgrade` command? Here's how to resolve this

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#604] [General]
If you're just starting out and have downloaded the meta pax in the last few days, then this issue likely does not affect you.

We recently detected a bug with the zopen JSON API processing. This symptom is that zopen upgrade will not detect new releases. 
The bug is fixed in a newer meta release, but in order to obtain the newer meta release, you will need to follow these steps:

```
# Delete the zopen_releases json cache
rm -f ${ZOPEN_ROOTFS}/var/cache/zopen/zopen_releases*
# Upgrade meta
zopen upgrade meta -y
# Upgrade your tools:
zopen upgrade -y
```

**Comment by AnthonyGiorgio** [comment]
That worked perfectly.  Thanks!
